staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/config_test.go

```diff
@@ -56,7 +49,7 @@ resources:
           secret: dGhpcyBpcyBwYXNzd29yZA==
     - kms:
         name: testprovider
-        configfile: testproviderconfig
+        endpoint: /tmp/testprovider.sock
```

Comment:
nit, make the endpoints in the test data actually be valid (`unix:///...`)

Reply:
Addressed